packages/skatejs/test/unit/with-update.spec.js

```diff
@@ -98,6 +98,9 @@ test('prop types could be defined with native types constructors', () => {
   class Test extends withUpdate() {
     static get props() {
       return {
+        propsAny: props.any,
```

Comment:
I'm actually fine if we assert something like:

```js
expect(Component.props.nativeNull).toMatchObject(props.any);
```

And remove direct testing of `props.any` in withUpdate (and of course continue testing them separately as a unit).

Reply:
I'd also like to start advocating the usage of `null`, `String`, `Array`, `Object`, etc over the `props.*` as it's definitely closer to the metal in how one might expect it to behave. Custom usage would involve the `props.*` API.

Reply:
This might be out of the scope of this PR and we can do it in 6, though. For now, I think I'm happy to see this as is, but your thoughts here would be useful.

Reply:
> expect(Component.props.nativeNull).toMatchObject(props.any);

As far as i remember this did not work because of the prop name saved in prop definition. I can try again and put the error here

> And remove direct testing of props.any in withUpdate (and of course continue testing them separately as a unit).

Do you mean creating a separated spec file?

> This might be out of the scope of this PR and we can do it in 6, though. For now, I think I'm happy to see this as is, but your thoughts here would be useful.

Agree

Reply:
I'm happy to keep as-is for the short-term.